package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node test/helpers/pretest.js",
```

Comment:
why pretest? you should use the funcs directly in the "before" hooks of the mocha tests you want to clean up/create the sandbox for.